smoke-tests/spec/external_dns_helper.rb

```diff
@@ -1,3 +1,15 @@
+# Return the zone_id of a zone, found by name (domain)
+# This will only use the first result
+def get_zone_by_name(domain)
+  client = Aws::Route53::Client.new
+
+  zones = client.list_hosted_zones_by_name({
+    dns_name: domain,
+    max_items: 1
+  })
+  zone_id = zones.hosted_zones[0].id.tr("/hostedzone/", "")
```

Comment:
You don't need the "zone_id = " here - whatever the last value evaluated in the function is, that's what the return value will be.